pkg/kudoctl/kudoinit/manager/manager.go

```diff
@@ -32,6 +32,8 @@ type Initializer struct {
 	deployment *appsv1.StatefulSet
 }
 
+const Name = "manager"
```

Comment:
We started to consolidate all the hard-coded names in [types.go](https://github.com/kudobuilder/kudo/blob/main/pkg/kudoctl/kudoinit/types.go#L12). Please move this one there too

Reply:
I much prefer it here... packages communicate good meaning and `manager.Name` is very clear IMO

Reply:
just reviewed types.go...  not a fan of what is there.  for instance ManagerName is not true... that is the manager pod name.  IMO it would be better to have `manager.PodName` than `kudoinit.DefaultManagerName` which isn't correct...  Based on this assertion... perhaps `manager.Name` could be more accurate as `manager.ContainerName` but it seems like a reasonable abbrev.  It seems to we that all other uses of `types.go` in the maturer kube eco-system is for values used in the API.  For kudoinit, that would be types / options used in Artifacter or InstallVerifier, Step, etc.   The constants defined here now, do not adhere to that and thus seem off to me.  Without tribal nugging  I wouldn't know or to think to put them there.   What is the position in having them there?

Reply:
It's subjective of course but I do like `kudoinit.DefaultManagerName` more than `manager.Name`. First clearly indicates that this is for the _init_ while the second could be anything manager related. I agree that some naming can be improved but my main argument is about consistency: we can have these constants one way (all in one place) or another (each in its own package) but please not both.

Reply:
agree on subjectivity...  part of my questioning is the scenario you are bringing up...  for the current use case this is _init_ so `kudoinit` makes sense... but it is likely that we would use these constants in other ways which isn't init...  By defining them in the package for which they define state they can be leverage by packages that define behavior (if it is a separate package).  this doesn't work well the other way IMO.   
It would be great to meet around some standards and conventions as well.  It would be good to get more aligned on this...  having constants "all in one place" isn't a goal that makes sense to me... at least not in isolation.